src/components/pages/AddJob.vue

```diff
@@ -21,6 +21,8 @@ export default {
       isPreview: false,
       isSaved: false,
       isSaving: false,
+      span_tags: [],
```

Comment:
`spanTags` is better I think :) Because camelcase naming convention is used instead of underscore